go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/containerd/containerd v1.3.2 // indirect
 	github.com/containerd/fifo v0.0.0-20190226154929-a9fb20d87448 // indirect
 	github.com/coreos/go-systemd v0.0.0-20190321100706-95778dfbb74e
-	github.com/cortexproject/cortex v0.7.0-rc.0
+	github.com/cortexproject/cortex v0.6.1-0.20200312144321-c105d22e5ad0
```

Comment:
is this going down a revision because this is current master vs whatever is in 0.7.0rc?

Reply:
Go finds latest version tag reachable from commit, and uses that as version number. 0.7* tags are not reachable from master, since release branch wasn't merged to master yet.

Reply:
Do we have any guarantees we don't rely on something from `0.7.0-rc.0`?

Reply:
I've pushed new update that updates this to 0.7.1-* now. It is master, with release-0.7 branch merged in.